### PR TITLE
16_As a user, I need to get the RSS flow

### PR DESCRIPTION
**Pull request by Jean-Beru**
Issue #1

### CODE REVIEW

File: src/SensioLabs/JobBoardBundle/Controller/BaseController.php [hunk 1]
```diff
@@ -59,4 +67,28 @@ public function manageAction(Request $request)
...
+     */
+    public function feedAction(Request $request)
+    {
+        $filters = [
+            'country' => $request->query->get('country'),
+            'contract-type' => $request->query->get('contract-type'),
+            'restrictToVisible' => true,
+            'status' => Job::STATUS_PUBLISHED,
+        ];
+        $orders = [
```
Comment (salahm) [inline src/SensioLabs/JobBoardBundle/Controller/BaseController.php]: there is no really need here to have an array in multiple lines

File: src/SensioLabs/JobBoardBundle/Entity/JobRepository.php [hunk 2]
```diff
@@ -13,18 +13,22 @@ class JobRepository extends EntityRepository
...
      */
-    public function getFiltered($filters = [], $page = 1, $limit = self::DEFAULT_HOMEPAGE_JOBS_PER_PAGE)
+    public function getFiltered($filters = [], $orders = [], $page = null, $limit = null)
     {
-        $qb = $this->createQueryBuilder('j')
-            ->innerJoin('j.company', 'c')
-            ->setFirstResult(($page - 1) * $limit)
-            ->setMaxResults($limit);
+        $qb = $this->createQueryBuilder('j')->innerJoin('j.company', 'c');
+        if ($page !== null && $limit !== null) {
```
Comment (salahm) [inline src/SensioLabs/JobBoardBundle/Entity/JobRepository.php]: please use Yoda notations (https://en.wikipedia.org/wiki/Yoda_conditions) for comparaison

File: src/SensioLabs/JobBoardBundle/Controller/BaseController.php [hunk 3]
```diff
@@ -58,4 +66,26 @@ public function manageAction(Request $request)
...
+            'status' => Job::STATUS_PUBLISHED,
+        ];
+        $orders = ['visibleFrom' => 'DESC'];
+        $repo = $this->getDoctrine()->getManager()->getRepository('SensioLabsJobBoardBundle:Job');
+        $jobs = $repo->getFiltered($filters, $orders);
+
+        $feed = $this->get('eko_feed.feed.manager')->get('jobs');
+        $feed->addFromArray($jobs);
+
+        return new Response($feed->render('rss'));
```
Comment (ogizanagi) [inline src/SensioLabs/JobBoardBundle/Controller/BaseController.php]: Please, can you ensure the `Content-Type` is right ?

**Comment by ogizanagi** [comment]
Appart from my comment, :+1:

File: src/SensioLabs/JobBoardBundle/Controller/BaseController.php (hunk 3 above)
Comment (Jean-Beru) [inline src/SensioLabs/JobBoardBundle/Controller/BaseController.php]: Done. Content-type ensured and tested ;-)

**Pull request merged**